### ISSUE TITLE
[Request] Get "a" graph definition json

### ISSUE THREAD

**Issue by budougumi0617** [a-know/Pixela#15]
I appreciate excellent product. Today, I have a request.

I want an endpoint that returns JSON for "a" graph for a specified ID.

Details:
Now I'm implementing a terraform provider for Pixela.

https://github.com/budougumi0617/terraform-provider-pixela

To create a terraform provider, we need to be able to perform CRUD operations to the resources. 

https://learn.hashicorp.com/tutorials/terraform/provider-use?in=terraform/providers

However, your service does not have an public endpoint to get the json of the definition for the specified graph.
`GET - /v1/users/<username>/graphs/<graphID>` returns svg data.
- https://docs.pixe.la/entry/get-svg

Therefore, my implementation is using an alternative what I [get the JSON for all the graph definitions](https://docs.pixe.la/entry/get-graph) and then search the graph information for a specified ID. even as the number of graphs defined by provider increases, this operation will be performed for each graph.
This is an unnecessary operation and may put an extra load on the server.

I'd appreciate if you would examine this request.

Regards,

**Comment by a-know** [comment]
@budougumi0617 san,
Thank you for your request! 😄 
I have just released `v1.21.0` that includes the requested API addition!

https://github.com/a-know/Pixela/releases/tag/v1.21.0

If this will satisfy your request, please close this issue.
I'm looking forward to your request anytime! 😄

**Comment by budougumi0617** [comment]
Thank you for your fast release! I got below response.
```json
curl -X GET "https://pixe.la/v1/users/budougumi0617/graphs/egiu/graph-def" -H "X-USER-TOKEN:${PIXELA_TOKEN}" | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   203  100   203    0     0    652      0 --:--:-- --:--:-- --:--:--   652
{
  "id": "egiu",
  "name": "English grammar in use",
  "unit": "unit",
  "type": "int",
  "color": "ajisai",
  "timezone": "Asia/Tokyo",
  "purgeCacheURLs": null,
  "selfSufficient": "none",
  "isSecret": true,
  "publishOptionalData": false
}
```

this is my wanted endpoint, thanks!!
